### PR TITLE
Hungarian localization

### PR DESCRIPTION
**Pull request by KAMI911**
Started 2 localization tasks

### CODE REVIEW

File: WindowListGroup@jake.phy@gmail.com/files/WindowListGroup@jake.phy@gmail.com/locale/hu.po [hunk 1]
```diff
@@ -0,0 +1,340 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (jaszhix) [inline WindowListGroup@jake.phy@gmail.com/files/WindowListGroup@jake.phy@gmail.com/locale/hu.po]: Can you delete this file from this PR and submit it as a separate PR? Please see [contributing](https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.github/CONTRIBUTING.md), and update the PR title.

**Comment by KAMI911** [comment]
Commits have separated. I hope PR is  better now.

**Comment by jaszhix** [comment]
Supposed to only change one applet per PR, unless the changes are similar but since you are translating two different applets they should be in separate PRs. 

> Can you delete this file from this PR and submit it as a separate PR?

**Comment by jaszhix** [comment]
Merged #1789, #1790. Thanks!
